aspnetcore/fundamentals/logging/index.md

```diff
@@ -816,6 +816,19 @@ The `Logging` property is of type <xref:Microsoft.Extensions.Logging.ILoggingBui
 
 Logging configuration can be loaded from app settings files. For more information, see <xref:blazor/hosting-model-configuration#logging-configuration>.
 
+#### Blazor WebAssembly SignalR .NET client logging
+
+Inject an instance of <xref:Microsoft.Extensions.Logging.ILoggerProvider>, `loggerProvider` in the following example, and call <xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilderExtensions.ConfigureLogging%2A> to add the provider:
```

Comment:
>Inject an instance of <xref:Microsoft.Extensions.Logging.ILoggerProvider>

Is it obvious how to do that, or should it be flushed out more?

Reply:
Is the injection is done differently depending on where/how this code appears (e.g., in a ctor vs. not in a ctor)? I was thinking that if injection is flexible that this would be written in this general way to cover multiple scenarios.

Reply:
No ... rubber 🦆 says "NavigationManager" is the tip off ... it's the normal component injection.

I'll fix it on the next commit.